config/gcp.spc

```diff
@@ -19,4 +19,8 @@ connection "gcp" {
   # `impersonate_service_account` (optional) - The GCP service account (string) which should be impersonated.
   # If not set, no impersonation is done.
   #impersonate_service_account = "YOUR_SERVICE_ACCOUNT"
+
+  # List of additional GCP error codes to ignore for all queries.
+  # By default, common not found error codes are ignored and will still be ignored even if this argument is not set.
+  #ignore_error_codes = ["AccessDenied", "AccessDeniedException", "NotAuthorized", "UnauthorizedOperation", "AuthorizationError", "403"]
```

Comment:
@ParthaI Are all the above error codes returned by the GCP API?

Reply:
No, I have just given the examples, should we update it as per API error?

Reply:
@ParthaI we should update it as per the API error.